guides/source/developers/extensions/decorators.html.md

```diff
@@ -32,6 +32,34 @@ total by $10 (or whatever your currency is).
 
 [monkey-patch]: https://en.wikipedia.org/wiki/Monkey_patch
 
+## Using ActiveRecord methods in decorators
```

Comment:
I'd not say `ActiveRecord methods` here, they are more generic class-level methods. For example, the `acts_as_something` methods coming from other libraries are not `ActiveRecord`.

Reply:
Changed to class-level, does that make sense?